hq/app/views/fragments/openSecurityGroups.scala.html

```diff
@@ -59,8 +61,8 @@
 
                         </tr>
                         <tr>
-                            <th>VPC</th>
-                            <td>@flaggedSg.vpcId</td>
+                            <th>VPC Id/Name</th>
+                            <td>@flaggedSg.vpcId  / @flaggedSg.vpcName.getOrElse("N/A")</td>
```

Comment:
Could we make the name a new row in the table, so we have separate rows for the VPC ID and VPC name?

This will give us less layout issues, and make the table content a bit clearer to new users.